apps/api/src/schema/websoc.ts

```diff
@@ -75,18 +75,36 @@ const isValidRestrictionCode = (code: string): code is (typeof restrictionCodes)
 export const websocQuerySchema = z.object({
   year: z
     .string({ message: "Parameter 'year' is required " })
-    .length(4, { message: "Parameter 'year' must have length 4" }),
-  quarter: z.enum(terms, { required_error: "Parameter 'quarter' is required" }),
+    .length(4, { message: "Parameter 'year' must have length 4" })
+    .openapi({ description: "The academic year of the course ", example: "2025" }),
+  quarter: z.enum(terms, { required_error: "Parameter 'quarter' is required" }).openapi({
+    description: "The academic quarter of the course",
+    example: "Fall",
+  }),
   ge: z
     .enum(geCategories)
     .optional()
+    .openapi({ description: "The general education category of the course", example: "GE-1A" })
     .transform((x) => (x === "ANY" ? undefined : x)),
-  department: z.string().optional(),
-  courseTitle: z.string().optional(),
-  courseNumber: courseNumberSchema.optional(),
+  department: z.string().optional().openapi({
+    description: "The department code of the course",
+    example: "I&C SCI",
+  }),
+  courseTitle: z.string().optional().openapi({
+    description: "The course title of the course",
+    example: "DATA STRC IMPL&ANLS",
+  }),
+  courseNumber: courseNumberSchema.optional().openapi({
+    description: "The course number(s) of the course",
+    example: "46",
+  }),
   sectionCodes: z
     .string()
     .optional()
+    .openapi({
+      description: "The section code of the course",
```

Comment:
Look at the code just below this patch and the choice of plural in `sectionCodes`. This parameter does more than getting exactly one section code. Describe that behavior sufficiently for a specification.

Also note that section codes describe sections, not codes.